Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -965,23 +965,6 @@ public function __construct()
         $this->makeTest($expected);
     }
 
-    public function testStatic()
-    {
-        $expected = <<<'EOF'
-<?php
-
-static class Foo
-{
-    static function Foo()
```

Comment:
Yes, indeed, there are not static constructors. So this is only a static function. And the test is to prove that function called after class (so PHP4 constructor?) is not a constructor because it is static function, therefore is not changed into PHP5 constructor ;)